<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QMainWindow" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>32</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="user_radioButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>注册用户</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="admin_radioButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>注册管理员</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>UID：</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QLineEdit" name="uid_lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>18</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用户姓名：</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QLineEdit" name="name_lineEdit_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>18</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="register_pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
